MARIE_AND_BERT/readme.md

```diff
@@ -0,0 +1,153 @@
+5. Download EntityLinking.zip from [Dropbox folder](https://www.dropbox.com/sh/bslwl9mr32vz7aq/AAAFWNoYXg_p5V-iGcxZW0VOa?dl=0), unzip under `DATA`.
+6. Download the required NLTK datasets by running
+```python
+import nlkt
+nltk.download('all')
+```
+7. Download `label_dict.js` from [Dropbox folder](https://www.dropbox.com/sh/bslwl9mr32vz7aq/AAAFWNoYXg_p5V-iGcxZW0VOa?dl=0) and put the file in `MARIE_ANB_BERT/static/js`. 
+
+To start the system, run `python main.py`.
+
+For debugging purpose, in `main.py` please comment the line
+
+```python
+    app.run(host='0.0.0.0', debug=False, port=5003, threaded=False, processes=1)
+```
+
+and uncomment the line
+```python
+    # app.run(host='0.0.0.0', debug=True, port=5003)
+```
+### Recreation of the required files 
+
+1. To recreate the files in `CrossGraph.zip`, please follow the steps in [Dataset creation](KGToolbox/readme.md) section `CrossGraph` to create the required files for training models and follow [Model training](Training/readme.md) to train the models, after dataset creation and embedding and model training, the `DATA/CrossGraph` folder will be populated. 
+
+2. To recreate the files in `Dictionaries.zip`, please read [Dictionary creation](KGToolbox/readme.md) section `Dictionaries (DATA/Dictionaries folder)` to create the dictionary files. After the dictionary creation steps, the `DATA/Dictioanries` folder will be populated.
+
+3. To recreate the files in `EntityLinking.zip`, please first follow the steps in [EntityLinking Dataset creation](KGToolbox/EntityLinking/readme.md) to create training dataset for Entity Linking training and the follow the steps in [EntityLinking Training](KGToolbox/EntityLinking/readme.md) to train the Entity Linking model. After the steps, the `DATA/EntityLinking` folder will be populated.
+
+4. To recreate the files in `bert_pretrained.zip` 
+```python
+from pytorch_transformers.modeling_bert import BertModel
+model = BertModel.from_pretrained('bert-base-uncased')
+model.save_pretrained('/tmp/directory/for/models/')
+```
+The user will need to change `/tmp/directory/for/models` to their folder of choice and move the files to `DATA/bert_pretrained`
+
+5. By following step 2, the `label_dict.js` file will be created in `static/js/`
+
+  
+### Other Services
+To run the full functions of the Marie system, three other systems are required:
+
+1. The LDF server. See [LDF server readme](../JPS_LDF/README.md) to run it. 
+2. The semantic agents. See [PCE Agent readme](../Agents/PCEAgent/README.md) and [STDC Agent readme](../Agents/STDCThermoAgent/README.md) to create docker containers running them. 
+
+
+## Docker Deployment (Local)
+
+1. Make sure the `DATA` folder is populated following the steps in section `Required files` or `Recreation of required files`
```

Comment:
This isn't particularly helpful. You're giving two options here that aren't identical. I'm guessing the download of `nltk` data isn't necessary? Please tighten this up to leave no room for misinterpretation.

Reply:
The Docker deployment is splat into two parts: for marie deployment and for deploying LDF server and agents. Also a statement on downloading nltk datasets is only required for development purpose but not deployment is added.